eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -100,16 +100,17 @@ jobs:
 
         - template: /eng/common/pipelines/templates/steps/set-default-branch.yml
 
-        - pwsh: |
-            az login
-            az artifacts universal download `
-            --organization https://dev.azure.com/azure-sdk/ `
-            --project="public" `
-            --scope project `
-            --feed azure-sdk-for-cpp `
-            --name my-first-package `
-            --version 0.0.1 `
-            --path .     
+        - task: AzurePowerShell@5
```

Comment:
For the download you should be able to just use the pwsh task without a service connection.